typescript/phoenix-sdk/src/client.ts

```diff
@@ -167,6 +167,45 @@ export class Client {
     });
   }
 
+  static async createWithoutConfig(
+    connection: Connection,
+    marketAddresses: PublicKey[],
+    trader?: PublicKey
+  ): Promise<Client> {
+    const accounts = await connection.getMultipleAccountsInfo(
+      [...marketAddresses, SYSVAR_CLOCK_PUBKEY],
+      "confirmed"
+    );
+
+    const clockBuffer = accounts.pop()?.data;
+    if (clockBuffer === undefined) {
+      throw new Error("Unable to get clock");
+    }
+    const clock = deserializeClockData(clockBuffer);
+
+    const marketKeysToData: Array<[PublicKey, AccountInfo<Buffer>]> =
```

Comment:
this field is misleadingly named; can we give it a better name, change it here, and also change it in `loadMarketsAndTokens`? name implies it's a map